content/projects/gsoc/students.adoc

```diff
@@ -29,16 +29,19 @@ It is required for the initial review and feedback collection.
 
 === First email to jenkinsci-dev
 
-* Please use the _[GSoC]_ prefix in your message subject
-* In the first e-mail we would be interested to see the following information:
- * A short self-introduction: your area of study, interests, background
- * Motivation letter. Why are you interested in the Jenkins project? On which project areas would you like to work? If there are particular proposals, please let us know about them as well, and any initial thoughts on why you would be suited
- * If you participate in open-source projects, please reference them
- * If you have profile pages in professional networks like LinkedIn, please reference them
- * If you have a Twitter account, a blog or technical/scientific publications, please reference them as well
+* Selecting a thread subject
+** Please use the _[GSoC]_ prefix in your email thread subjects.
+** It is recommended to introduce yourself in a project idea thread if it already exists.
+** Creating personal personal intro thread is also fine,
```

Comment:
personal personal


I also suggest nesting this under the bullet point above it.